monai/metrics/morphological_hausdorff_distance.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+from typing import Optional, Union
+
+import numpy as np
+import torch
+from monai._extensions.loader import load_module
+from .metric import CumulativeIterationMetric
+
+__all__ = ["MorphologicalHausdorffDistanceMetric"]
+
+
+class MorphologicalHausdorffDistanceMetric(CumulativeIterationMetric):
```

Comment:
Should this class work like `HausdorffDistanceMetric` with a `_compute_tensor` method?